src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -60,6 +60,9 @@
     public static final int MAX_RECORDS_IN_RAM = 500000;
 
     private static final char[] RESERVED_ATTRIBUTE_STARTS = {'X', 'Y', 'Z'};
+    static final String HARD_CLIPPED_BASES_TAG = "hB";
```

Comment:
since soft-clipped bases are not removed, I think that the "h" is superfluous. how about EB and EQ (for End Bases, and End Quality) since in theory, bases could also be (hard-)clipped off of the start of a read..

Reply:
also, as a //Todo item here to switch to using the htsjdk version if it gets into hts-spec and implemented